data/ka'het ships.txt

```diff
@@ -0,0 +1,322 @@
+	description "The Maeri is the smallest Het ship created by the Builders' stations, a prime example of the genetic technology they once possessed. Its small size is compensated by the fact that it can hide its location from most types of sensors."
+
+ship "Maeri'het" "Maeri'het (Disable-able)"
+	"uncapturable"
+
+ship "Maeri'het" "Maeri'het (EMP)"
+	"never disabled"
+	outfits
+		"EMP Torpedo" 32
+		"Ka'het Ravager Turret"
+		"Ka'het EMP Deployer" 2
+		"Ka'het Annihilator Turret" 3
+
+		"Ka'het Support Cooling"
+		"Ka'het Primary Cooling"
+		"Ka'het Shield Restorer" 2
+
+		Hyperdrive
+
+ship "Maeri'het" "Maeri'het (EMP, Disable-able)"
+	"uncapturable"
+	outfits
+		"EMP Torpedo" 32
+		"Ka'het Ravager Turret"
+		"Ka'het EMP Deployer" 2
+		"Ka'het Annihilator Turret" 3
+
+		"Ka'het Support Cooling"
+		"Ka'het Primary Cooling"
+		"Ka'het Shield Restorer" 2
+
+		Hyperdrive
+
+ship "Telis'het"
+	plural "Telis'heta"
+	sprite "ship/telis'het"
+	"never disabled"
+	attributes
+		category "Heavy Warship"
+		"cost" 32600000
+		"shields" 47000
+		"hull" 26300
+		"required crew" 1
+		"bunks" 1
+		"mass" 620
+		"drag" 6.9
+		"heat dissipation" .35
+		"fuel capacity" 800
+		"ramscoop" 0.82
+		"cargo space" 71
+		"outfit space" 513
+		"weapon capacity" 254
+		"energy capacity" 28000
+		"energy generation" 37.4
+		"heat generation" 40
+		"hull repair" 0.5
+		"shield energy" 0.3
+		"shield heat" 9.2
+		"thrust" 62.8
+		"thrusting energy" 11.4
+		"thrusting heat" 3.5
+		"flare sprite" "effect/remnant flare/small"
+			"frame rate" 5
+		"flare sprite" "effect/remnant flare/medium"
+			"frame rate" 4
+		"flare sound" "plasma medium"
+		"turn" 2031.7
+		"turning energy" 6.38
+		"turning heat" 2.9
+		weapon
+			"blast radius" 300
+			"shield damage" 1200
+			"hull damage" 600
+			"hit force" 1800
+	outfits
+		"Ka'het Annihilator Turret" 2
+		"Ka'het EMP Deployer" 2
+		"Ka'het Ravager Turret" 3
+		"EMP Torpedo" 32
+		"Ka'het Annihilator" 2
+
+		"Ka'het Shield Restorer" 7
+		"Ka'het Primary Cooling" 2
+
+		"Hyperdrive"
+
+	engine -27 75.5 0.6
+	engine -39.5 82.5 0.8
+	engine 27 75.5 0.6
+	engine 39.5 82.5 0.8
+	gun -11.5 -94 "Ka'het EMP Deployer"
+	gun -23.5 -87.5 "Ka'het Annihilator"
+	gun 11.5 -94 "Ka'het EMP Deployer"
+	gun 23.5 -87.5 "Ka'het Annihilator"
+	turret -27 -41 "Ka'het Annihilator Turret"
+	turret -41.5 -50.5 "Ka'het Ravager Turret"
+	turret 0 -53 "Ka'het Ravager Turret"
+	turret 27 -41 "Ka'het Annihilator Turret"
+	turret 41.5 -50.5 "Ka'het Ravager Turret"
+	explode "small explosion" 25
+	explode "medium explosion" 30
+	explode "large explosion" 20
+	explode "huge explosion" 5
+	"final explode" "final explosion large"
+	description "When the Builders designed the exoskeleton of the Telis, they came to the conclusion that it would have been impossible to make it resilient enough without compromising its agility and integrity. Instead, they focused heavily on improving their shield technology, giving the Telis many of their regenerators."
```

Comment:
The Telis goes 95% the max speed and acceleration of the Maeri, and actually has 22% better steering, so I'm not so sure that the Telis has really lost in the agility department as a result of being bigger.